userguide/sharing.rst

```diff
@@ -1006,135 +1006,142 @@ for more details.
+   | syncops              | Ensure metadata operations are performed synchronously.                                                                         |
+   |                      |                                                                                                                                 |
+   +----------------------+---------------------------------------------------------------------------------------------------------------------------------+
+   | time_audit           | Log system calls that take longer than the defined number of milliseconds.                                                      |
+   |                      |                                                                                                                                 |
+   +----------------------+---------------------------------------------------------------------------------------------------------------------------------+
+   | unityed_media        | Allow multiple Avid clients to share a network drive.                                                                           |
+   |                      |                                                                                                                                 |
+   +----------------------+---------------------------------------------------------------------------------------------------------------------------------+
+   | virusfilter          | This extremely **experimental** module is still under development and does not work at this time.                               |
+   +----------------------+---------------------------------------------------------------------------------------------------------------------------------+
+   | winmsa               | Emulate the Microsoft *MoveSecurityAttributes=0* registry option. Moving files or directories sets the ACL for file and         |
+   |                      | directory hierarchies to inherit from the destination directory.                                                                |
+   +----------------------+---------------------------------------------------------------------------------------------------------------------------------+
+   | worm                 | Control the writability of files and folders depending on their change time and an adjustable grace period.                     |
+   |                      |                                                                                                                                 |
+   +----------------------+---------------------------------------------------------------------------------------------------------------------------------+
+   | xattr\_\ :file:`tdb` | Store Extended Attributes (EAs) in a tdb file so they can be used on filesystems which do not provide support for EAs.          |
```

Comment:
The first tdb is part of a module name and should not have markup.  The second tdb should have :file: markup, as it did in the other version of this patch.